packages/integration-tests/src/integrationTests/registrar.test.ts

```diff
@@ -72,18 +71,11 @@ describe('registrar is offline, user tries to join, then registrar goes online',
       registrarAddress
     })
     // User should keep sending requests for 10 seconds.
-    await sleep(10_000)
+    await assertRegistrationRequestSent(user.store, 2)
```

Comment:
How do you know that it will attempt to register exactly 2 times?

Reply:
expect(store.getState().Communities.communities.entities[communityId].registrationAttempts).toEqual(count)
  }, 240_000)
}

Reply:
I've moved registrationAttempts to store, so we can track that